info.plist

```diff
@@ -71,15 +71,45 @@
 				<key>runningsubtext</key>
 				<string>Searching...</string>
 				<key>script</key>
-				<string>eval $(/usr/libexec/path_helper -s)
-eval "$($SHELL -i -l -c 'echo -e "\n"PATH=\"$PATH:\$PATH\""\n"' 2&gt;/dev/null | grep "^PATH=")"
-export PATH
+				<string>PATH_CACHE=.path_cache
```

Comment:
Alfred has a dedicated cache directory you can get with the `alfred_workflow_cache` environment variable.

https://www.alfredapp.com/help/workflows/script-environment-variables/